experimental/class-access-expressions.md

```diff
@@ -0,0 +1,23 @@
+# [class property access expressions][proposal-class-access-expressions]
+
+## ClassObject
+
+```js
+interface ClassObject <: Node {
+    type: "ClassObject";
```

Comment:
The `ClassObject` is meant to be aligned with `Super`. Unfortunately we can not name it `Class` because `Class` has been used for class declarations / expressions, which is more common.

I pick `ClassObject` from the spec text. Any naming suggestion is welcome.